falcon/testing/client.py

```diff
@@ -2284,6 +2317,13 @@ def _prepare_sim_args(
         headers = dict(headers or {})
         headers['Content-Type'] = MEDIA_JSON
 
+    if msgpack is not None:
+        body = MessagePackHandler.serialize(
+            MessagePackHandler(), content_type=None, media=msgpack
```

Comment:
This is a rather strange form to call `MessagePackHandler`, wouldn't it work just as a simple invocation below?
```python
MessagePackHandler().serialize(content_type=None, media=msgpack)
```

Reply:
Fixed that!